src/__init__.py

```diff
@@ -0,0 +1,14 @@
+#  Copyright 2020 Waterball (johnny850807@gmail.com)
```

Comment:
Let's move the test to `./tests`.
Also, do you want to unify the header? not sure everyone installs the same plugin and has the same config or not.
Besides this, maybe you would like to use something like `spic-committee`. Otherwise, you will see ppl's email here and there. And ppl can be listed README, where should have a section like contributors, should be good enough.

Reply:
I think this part is only related to LICENSE, and I'm as the LICENSE owner also Apache LICENSE requires my signature there. I will put contributors including all of us in the README then. 👍 

What is `spic-committee` btw? Couldn't find information about it.